### ISSUE TITLE
offline messages batching

### ISSUE THREAD

**Issue by abdelhameedhamdy** [mqttjs/MQTT.js#1699]
Hey, Greetings,

I have a subscriber with qos 1 and clean set to false, to fetch offline persisted messages from broker, while subscriber is offline.
I am wondering is there a way to process the received offline message as batches rather than one-by-one,

I think there is a inflight max messages count option in MQTT v5 protocol can be configured.

How could I enable this message batching technique using MQTT.js ? 

Thanks

**Comment by robertsLando** [comment]
I'm not aware of nothing on mqtt protocol side to handle sending batch offline messages. You could try by checking mqtt 5 specs.

**Comment by abdelhameedhamdy** [comment]
Theres is some thing called inflight messages count in mqtt 5 spec, https://docs.oasis-open.org/mqtt/mqtt/v5.0/mqtt-v5.0.html

**Comment by robertsLando** [comment]
Yeah but that I don't understand how you could use that to do what you are looking for. In order to do that you should implement some logic on broker side or on clients but I dunno how you implemented your MQTT application
